Simplified/NYPLBookRegistry.m

```diff
@@ -492,6 +492,12 @@ - (void)setState:(NYPLBookState)state forIdentifier:(NSString *const)identifier
   }
 }
 
```

Comment:
Please comment with TODO so we know that this is *because* of ObjC and can be removed when 100% swift

Reply:
Added comment